### PR TITLE
EDU-696 Add user profile page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-696

Screen Shot 2022-07-26 at 09 43 06

### CODE REVIEW

File: src/components/profile-tab.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 info:
-  en: Here you can provide additional information about yourself. This information is currently not publicly accessible.
-  de: Hier können Sie zusätzliche Informationen zu Ihrer Person angeben. Die Daten sind zur Zeit nicht öffentlich sichtbar.
+  en: Here you can provide additional information about yourself. This information is publicly accessible through your <0>profile page</0>.
+  de: Hier können Sie zusätzliche Informationen zu Ihrer Person angeben. Die Daten sind auf Ihren <0>Profilseite</0> öffentlich sichtbar.
```
Comment (ahelmberger) [inline src/components/profile-tab.yml:3]: Ihre**r**

File: src/components/pages/user.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+accountClosed:
+  en: This user account is closed.
+  de: Dieser Benutzerkonto ist geschlossen.
```
Comment (ahelmberger) [inline src/components/pages/user.yml:3]: Diese**s**

File: src/components/profile-tab.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/profile-tab.yml]: **Die** Seite, of course 🤦‍♀️ Such basic mistakes.. Corrected, thank you.

File: src/components/pages/user.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/pages/user.yml]: Ouch, another basic article mistake. Thanks, I also changed to "**was** closed" in English.

**Review by ahelmberger — APPROVED** [review]

**Comment by adrianaluchian** [comment]
> Looks very nice 😍

Thanks, still a pretty empty page but heading somewhere 😁

**Pull request merged**